### PR TITLE
Clean up Favorites Div on User Page

### PR DESCRIPTION
**Pull request by greywidget**

### CODE REVIEW

File: books/templates/user.html [hunk 1]
```diff
@@ -51,28 +51,22 @@
...
-              {% endfor %}
-            {% endfor %}
-          {% else %}
-            <p>No favorite books yet.</p>
-          {% endif %}
+{% if favorites %}
+  <div class="mui--text-display1">
+    Favorites
+  </div>
+  <div class="mui-appbar">
```
Comment (bbelderbos) [inline books/templates/user.html:58]: I don't like the blue too much, shall we remove this?

**Review by bbelderbos — CHANGES_REQUESTED** [review]
Awesome, shall we get rid of the blue (mui-appbar css class)?

**Comment by greywidget** [comment]
I agree. I've changed to class to **mui-text-display1**

**Comment by bbelderbos** [comment]
Thanks @greywidget

**Pull request merged**